docker-compose.dev.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+
+services:
+  app:
+    container_name: app
+    build:
+      context: .
+      dockerfile: ./docker/dev/Dockerfile
+    volumes:
```

Comment:
Do I understand correctly that iam-org-creator needs persistent storage attached and those are the mount paths?

Reply:
I don't think iam-org-creator needs persistent storage actually. I think those volumes are just to enable docker to use the node_modules&code without needing to `npm i` and `npm build`. But it looks like we're doing that anyways in the docker file, so I'm a bit confused. Maybe you can clarify @ahmedolaibrahim ?

Reply:
@jrhender yes that's correct. it to enable docker to access the node_modules and source code files. in the dev compose file, we do not run `npm build` so that's probably needed. as for node_modules, i think it's not  needed like you said since we also run `npm ci`